### PR TITLE
Add mobile-desktop version

### PR DESCRIPTION
**Pull request by hamzaw9**
In this pull request:

### Home Page:

- Created a header with a navigation bar and social media links.
- Created a dynamic "Leading Instructors" section.
- Added a main section with event details and a call-to-action button.
- Included a "Feature Courses" section showcasing different courses.
- Designed a "Partner" section displaying partner logos.
- Implemented a footer with the ITMinds logo and copyright information.

### About Page:

- Created a header with a navigation bar and social media links.
- Added a main section with event details and contact information.
- Included an "ITMinds Aim" section describing the organization's mission.
- Implemented a "See Previous Events" section with images and years of past events.
- Designed a "Partner" section displaying partner logos.
- Included a footer with the ITMinds logo and copyright information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,203 @@
...
+- **Responsive Design**
+- **DOM Manipulatio**
+- **Animations and Transitions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://hamzaw9.github.io/capstone-project-1/)
+
```
Comment (Qoosim) [inline README.md]: - [ ] KIndly look into this live demo link. It is not working properly. Below screenshot is what I got while opened the link.

### Live Demo Display

live-demo

File: index.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        <li><img src="images/partner5.svg" alt="Airbnb" /></li>
+      </div>
+    </section>
+
+    <!---------  Footer   ---------->
+
+    <footer class="footer">
+      <a href="#"><i class="fa-solid fa-microchip fa-2xl"></i> ITMinds</a>
+      <p>&copy; 2023 ITMinds. All rights reserved.</p>
+    </footer>
```
Comment (Qoosim) [inline index.html:154]: - [ ] Kindly reduce the padding top and bottom for this two sections. The parnter and the footer sections in order to match the guiding template. The content of your footer is not as plenty as what we have on the guiding template. Too much padding top and botton for this part is not necessary. Kindly reduce it with that of partner too. I understand that you're trying to make it to look exactly like guiding template which is what is expected from you. But that is too much in this regard. Kindly do just that.
- [ ] You're expected to produce a close replica of the design. This includes. font, color among others. Kindly reduce the opacity of the background color of your partner section so as to match the giving design. Your background color is too dark compare to the guiding temp. Kindly address that too.

### Your Design
your-footer-home

### Guiding Template
temp-footer-home

File: about.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+    <!---------  Event Section   ---------->
+
+    <section id="events" class="events event-about">
+      <h2>See Previous Events</h2>
+      <p>
+        Take a look at the past two Events which took place in Lahore and
+        Islamabad.
+      </p>
+      <div class="event-inner">
```
Comment (Qoosim) [inline about.html:87]: - [ ] KIndly change the dividing line color to the template color as to match the guiging template as required by the project requirements.
- [ ] Do not forget to change the color of `See Previous Event` to the color on the guiding template. This is to train ourselves on how to clients requests to the letter.

### Your Design
your-connect-line

### Guiding Template
temp-connect-line

File: index.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    <!---------  Feature Course Section   ---------->
+
+    <section id="courses" class="feature-courses">
+      <div class="feature-cour-heading">
+        <h2>Feature Courses</h2>
+        <hr />
+      </div>
+      <div class="course-container">
+        <div class="course">
+          <img src="images/program_icon_01.png" />
```
Comment (Qoosim) [inline index.html]: - [ ] I would like to recommend you to always include `alt` attribute inside you `img`  tag with appropriate text to help vision impaired readers. Kindly check other sections where the `img` tag is used and include the `alt` attribute as needed.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+          </div>
+          <div class="instructor-details">
+            <h3>Shadman Ali</h3>
+            <h4>Passionate Web Developer</h4>
+            <p>
+              Ability to simplify complex coding concepts, making learning
+              enjoyable.
+            </p>
+          </div>
+        </div>-->
```
Comment (Qoosim) [inline index.html]: - [ ] Let's always remove commented code. It does not make our codebase to look clean. Kindly remove this.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @hamzaw9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Great UI. :heavy_check_mark: 
-  Clean and clear codebase. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @hamzaw9, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**